install/tools/ipa-replica-conncheck

```diff
@@ -63,8 +63,7 @@ class SshExec(object):
     def __call__(self, command, verbose=False):
         # Bail if ssh is not installed
         if self.cmd is None:
-            root_logger.warning("WARNING: ssh not installed, skipping ssh test")
-            return ('', '', 0)
+            raise RuntimeError("ssh not installed, skipping ssh test")
```

Comment:
IMHO it would be a bit cleaner to move the whole check for `self.cmd` to the object constructor so that we can bail out as early as possible.